modules/virt-configuring-downward-metrics.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * virt/monitoring/virt-exposing-downward-metrics.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-downward-metrics_virt-using-downward-metrics"]
+= Configuring downward metrics
+
+You enable the capturing of downward metrics for a host VM by creating a configuration file that includes the following options:
+
+* A `downwardMetrics` device
+* A `virtio-serial` port, through which the metrics are exposed
```

Comment:
This is not required. Just the `downwardMetrics` device, which automatically creates the virtio-serial port.